### PR TITLE
Feat/user projects

### PR DESCRIPTION
**Pull request by stalhaali**
Implemented:

- Project section on user profile
- Add profile to navbar

### CODE REVIEW

File: Buddies.API/Controllers/ProfilesController.cs [hunk 1]
```diff
@@ -57,7 +57,36 @@ public async Task<ActionResult> GetProfile(int id)
                 profileResponse.Skills.Add(skill);
             }
 
-
+            //var userProjects = await _context.Projects.Where(p => p.Members.Contains(profile.User)).ToListAsync();
```
Comment (andy9775) [inline Buddies.API/Controllers/ProfilesController.cs]:
```suggestion
```

File: Buddies.API/Controllers/ProfilesController.cs [hunk 2]
```diff
@@ -57,7 +57,36 @@ public async Task<ActionResult> GetProfile(int id)
...
+            var allProjects = await _context.Projects
+                .Include(project => project.Members)
+                .Include(project => project.Owner)
+                .ThenInclude(owner => owner.Profile).ToListAsync();
+
+            //var userProjects = allProjects.Where(project => project.Owner == profile.User);
```
Comment (andy9775) [inline Buddies.API/Controllers/ProfilesController.cs]:
```suggestion
```

**Review by andy9775 — APPROVED** [review]
Just some minor nits, otherwise LGTM

File: Buddies.API/IO/UserProfileResponse.cs [hunk 3]
```diff
@@ -26,4 +26,7 @@ public record UserProfileResponse
 
     [Required]
     public List<SkillResponse> Skills { get; set; } = new List<SkillResponse>();
+
+    [Required(ErrorMessage = "")]
```
Comment (SyedTahaA) [inline Buddies.API/IO/UserProfileResponse.cs]:
```suggestion
    [Required]
```
empty error msg?

**Review by SyedTahaA — APPROVED** [review]

File: Buddies.Web/src/components/Navbar.tsx [hunk 4]
```diff
@@ -41,12 +41,17 @@ const Navbar: React.VFC = () => {
...
           {authState ? (
-            <AccountTab
-              authState={authState}
-              logout={() => {
-                router.push('/').then(() => logoutUser());
-              }}
-            />
+            <>
+
+              <Button color="inherit" sx={{ marginLeft: 'auto' }} href={`/Profiles/${parseInt(authState.nameid, 10)}`}>Profile</Button>
```
Comment (SyedTahaA) [inline Buddies.Web/src/components/Navbar.tsx]: Move this to AccountTab.tsx, since it should be in the dropdown.

**Review by SyedTahaA — APPROVED** [review]
lgtm

**Pull request merged**